test/unit/modules/transport.js

```diff
@@ -1028,550 +1220,1161 @@ describe('transport', () => {
+							expect(error).to.equal(null);
+							expect(result)
+								.to.have.property('peers')
+								.which.is.an('array').that.is.empty;
+							expect(result)
+								.to.have.property('success')
+								.which.is.equal(false);
+							expect(
+								modules.peers.shared.getPeers.calledWith({
+									limit: peersList.length,
+								})
+							).to.be.true;
+							return expect(modules.peers.list.called).to.be.false;
+						});
+					});
 
-				it(
-					'should return callback with result = {success: !err, peers: peers}'
-				);
+					it('should return callback with error = null and result = {success: true, peers: peers}', () => {
+						expect(error).to.be.equal(null);
+						expect(result)
+							.to.have.property('success')
+							.which.equals(true);
+						return expect(result)
+							.to.have.property('peers')
+							.which.is.an('array').that.is.not.empty;
+					});
+				});
 			});
 
 			describe('height', () => {
-				it('should call callback with error = null');
+				var currentHeight;
 
-				it(
-					'should call callback with result = {success: true, height: modules.system.getHeight()}'
-				);
+				beforeEach(done => {
+					currentHeight = 12345;
+					req = {};
+					modules.system.getHeight = sinonSandbox.stub().returns(currentHeight);
+					transportInstance.shared.height(req, (err, res) => {
+						error = err;
+						result = res;
+						done();
+					});
+				});
+
+				it('should call callback with error = null and result = {success: true, height: modules.system.getHeight()}', () => {
+					expect(error).to.be.equal(null);
+					expect(result)
+						.to.have.property('success')
+						.which.is.equal(true);
+					return expect(result)
+						.to.have.property('height')
+						.which.is.equal(currentHeight);
+				});
 			});
 
 			describe('status', () => {
-				it('should call callback with error = null');
+				var headers;
 
-				it(
-					'should call callback with result = {success: true, height: modules.system.getHeight(), broadhash: modules.system.getBroadhash(), nonce: modules.system.getNonce()}'
-				);
-			});
+				beforeEach(done => {
+					headers = {
+						height: 123,
+						broadhash:
+							'258974416d58533227c6a3da1b6333f0541b06c65b41e45cf31926847a3db1ea',
+						nonce: 'sYHEDBKcScaAAAYg',
+						httpPort: 8000,
+						version: 'v0.8.0',
+						os: 'debian',
+					};
+					req = {};
+					modules.system.headers = sinonSandbox.stub().returns(headers);
+					transportInstance.shared.status(req, (err, res) => {
+						error = err;
+						result = res;
+						done();
+					});
+				});
 
-			describe('postSignatures', () => {
-				describe('when query.signatures is defined', () => {
-					it('should call __private.receiveSignatures');
+				it('should call callback with error = null and result = {success: true, height: modules.system.getHeight(), broadhash: modules.system.getBroadhash(), nonce: modules.system.getNonce()}', () => {
```

Comment:
The test doesn't really correlate with the description (also it's pretty long).
So the description says: 
`should call .... with ... height: modules.system.getHeight()`, 
where the test checks the direct value
`expect(result).to.have.property('height').which.equals(headers.height);`
It should be either mocking `system.getHeight`, checking if it was called and if the value returned by it was assigned correctly or just align description to check if after headers are set up, the values are returned correctly:
```
describe('when headers are set')
   it('should call callback with result containing height = 123"')
```
Please align with other tests as well, and also separate checked values if you think it's relevant.